Frontend/src/views/MapView.vue

```diff
@@ -117,6 +128,32 @@ export default {
     createCluster(){
       this.markerCluster = new MarkerClusterer(this.map, this.markers, //Creates a cluster object which clusters all markers on the map
         {imagePath: 'https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/m'});
+    },
+    setupPointButton(addPointDiv){ //sets up the add point button

```

Comment:
Confused why this is a method, why isn't the point button in the template section with a styling id in the styles section. Can also add the vue-click method there as well

Reply:
Also if you had to create button by javascript, just assign a class to the button and have a styling class already created in styles

Reply:
Simple explanation: because google maps. It doesn't play well with components added on top of it... I improvised. I'll look into this further though to see if I can clean it up a lot because I agree that it clogs up this file.

Reply:
I fixed this. This function has been removed and the functionality is the same.